Foundation/URLSession/URLSession.swift

```diff
@@ -268,7 +268,7 @@ open class URLSession : NSObject {
     }
     
     open private(set) var delegateQueue: OperationQueue
-    open var delegate: URLSessionDelegate?
+    open private(set) var delegate: URLSessionDelegate?
```

Comment:
[`URLSession.delegate` is get-only in Darwin Foundation](https://developer.apple.com/documentation/foundation/urlsession/1411530-delegate#).

Therefore, we should be able to override it with a get-only property.